src/Microsoft.AspNet.Mvc.TagHelpers/Internal/PartialAttributeLoggerStructure.cs

```diff
@@ -65,8 +70,8 @@ public string Message
         public string Format()
         {
             var newLine = Environment.NewLine;
-            return
-                string.Format($"Tag Helper {_uniqueId} had partial matches while determining mode:{newLine}\t{{0}}",
+            return string.Format(
+                $"Tag Helper with ID {_uniqueId} in view '{_viewPath}' had partial matches while determining mode:{newLine}\t{{0}}",
```

Comment:
nit: this could be unrolled to avoid most of the additional `$"..."` / `string.Format(...)` nesting:
``` c#
  string.Format(
    "Tag Helper with ID {0} in view '{1}' had partial matches while determining mode:{2}\t{{3}}",
    _uniqueId,
    _viewPath,
    newline,
    string.Join($"{newLine}\t", _partialMatches.Select(partial => string.Format(
      "Mode '{0}' missing attributes:{1}\t\t{{2}} ",
      partial.Mode,
      newline,
      string.Join($"{newLine}\t\t", partial.MissingAttributes)))));
```